### ISSUE TITLE
Please don't invert init/start

### ISSUE THREAD

**Issue by thheller** [dazld/shadow-cljs-sass#1]
In general I recommend using the setup of `init` for one time initialization and `start/stop` for hot reload cycles. As described [here](https://code.thheller.com/blog/shadow-cljs/2019/08/25/hot-reload-in-clojurescript.html).

The [template](https://github.com/dazld/shadow-cljs-sass/blob/master/resources/leiningen/new/shadow_cljs_sass/core.cljs#L21-L28) flips this pattern. I would ask you to flip it to keep it inline with other articles.

```
(defn ^:dev/after-load start []
  (js/console.log "start")
  (r/render [app] root-element))

(defn init []
  (js/console.log "Init")
  (start))
```

**Comment by dazld** [comment]
great spot! that shouldn't be there! thank you for checking.

**Comment by dazld** [comment]
fixed on `0.1.1`
